### DISCUSSION TITLE
New LCD is it touch enabled?

### DISCUSSION THREAD

**Discussion by TommySharpNZ** [OpenEVSE/openevse_esp32_firmware#813] [Q&A]
It doesn't seem exactly clear if the new LCD screen supports touch? Can anyone confirm?

**Comment by chris1howell** [comment]
Touch is planned for the future. The current hardware has the footprint but
no chip populated. The current enclosure is has a transparent window and is
fully enclosed.

Plans are to add a cutout to expose the display, develop seals to keep
water out and add the chip.

We may even do a remote touch display.

**Comment by TommySharpNZ** [comment]
Doh, not sure why I didn't even think that for touchscreen we'd need a cutout LOL...
Having a remote display that could be in the kitchen would be pretty cool! One remote display for multiple chargers even cooler!
